specs/spring-cloud-contract-spec-java/src/main/java/org/springframework/cloud/contract/spec/internal/MessagingHeaders.java

```diff
@@ -20,16 +20,24 @@
  * Contains most commonly used messaging headers.
  *
  * @author Marcin Grzejszczak
+ * @author Tim Ysewyn
  * @since 1.1.2
  */
 public class MessagingHeaders {
```

Comment:
shouldn't this be `final` with a `private` constructor?

Reply:
Let me do the same like `RegexPatterns`, per your remark.